### DISCUSSION TITLE
Position of tool bar to show horizontal on the top of the map

### DISCUSSION THREAD

**Discussion by cubecleveland** [geoman-io/leaflet-geoman#1032] [General]
Is there any way to set the location of the toolbar to an aboslute position on the map> namely id like to pu it on the right top but in a horizontal direction?

**Comment by Falke-Design** [comment]
No it is not possible to set the Toolbar to an absolute position, they are default Leaflet Controls. But you can display it in a horizontal direction:

```
      .leaflet-pm-toolbar{
        display: flex;
      }

      .leaflet-pm-toolbar > .button-container a{
        border-right: 1px solid #ccc;
      }
```
